src/poffin.c

```diff
@@ -26,7 +28,7 @@ BOOL sub_0202A914(Poffin *poffin)
 
 void Poffin_clear(Poffin *poffin)
```

Comment:
```suggestion
void Poffin_Clear(Poffin *poffin)
```

Similarly (github will not let me do this on the associated lines):

```diff
-Poffin *Poffin_malloc(int heapID)
+Poffin *Poffin_New(int heapID)
```

```diff
-void Poffin_copy(Poffin *src, Poffin *dest)
+void Poffin_Copy(Poffin *src, Poffin *dest)
```